assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -479,6 +720,31 @@ describe("DiversionPage", () => {
     expect(screen.getByRole("button", { name: "Edit Detour" })).toBeVisible()
   })
 
+  test("'Share Detour Details' screen returns to editing screen when edit detour button is clicked", async () => {
```

Comment:
Any other validation needed here to ensure that returning to the Edit Detour page does indeed allow users to edit the detour? Maybe verify the undo and clear buttons are active again?

Reply:
I'm not opposed!

I think this is the sort of thing that shows my ... sort of point 2 of this type of work, that we may want to break this test file up into smaller ones that validate different use cases and/or different parts of the screen. Then it becomes clearer which bits of user-facing coverage are still missing, and makes the test files a lot less cluttered.

Reply:
[Feedback: Add assertions about undo and clear buttons when returning to edit screen](https://github.com/mbta/skate/pull/2580/commits/9cd5dcc5dca5519e09b529a91578004547a2864b)